src/Model.elm

```diff
@@ -7,4 +7,6 @@ model : Model
 model =
     { route = HomeRoute
     , userInput = ""
+    , focusedGroup = Nothing
+    , focusedOrganisation = Nothing
```

Comment:
TLDR; I tend to keep stuff like this (eg: `init`, `initialModel`, `update` and `subscriptions`) in one file called `State.elm` (you can call it `Update.elm` if that makes more sense), there's no need for it to be in its own file.

Not that you guys will likely have loads of time to watch this but it's an interesting vid: https://www.youtube.com/watch?v=XpDsk374LDE

Guy who made elm suggests that it's less important to keep files really small in elm and more important to organise them around certain data structures.

Conventional wisdom in JS for keeping files small is to limit "sneaky mutation", but this is impossible in elm (mutation can only happen outside your program).

Reply:
😍  OMG!! how lucky are we? @astroash @maxgerber these ara amazing tips! thanks so so much ❤️

Reply:
This is super, thanks @andrewMacmurray ! I've implemented these changes on the boilerplate https://github.com/astroash/elm-spa-boiler-plate/pull/12

If you get a chance can you check this is right so that our BP improved moving forward ❤️

Reply:
@rebecacalvoquintero you are cracking me up 😂 glad they are of some use. @astroash sure, will have a look